pkg/apis/servicecatalog/types.go

```diff
@@ -32,8 +32,9 @@ type ClusterServiceBroker struct {
 	metav1.TypeMeta
 	metav1.ObjectMeta
 
-	Spec   ClusterServiceBrokerSpec
-	Status ClusterServiceBrokerStatus
+	Spec           ClusterServiceBrokerSpec
+	Status         ClusterServiceBrokerStatus
+	LastReconciled metav1.Time
```

Comment:
This field should be part of status and should be called LastCatalogRetrievalTime, and should be a pointer (zero value time is not meaningful)